docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "59db66ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture \n",
+    "!pip install transformers tqdm "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "e20de927",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# packages version: \n",
+    "# numpy == 1.23.0 \n",
+    "# tqdm == 4.64.0 \n",
+    "# transformers == 4.20.1 "
```

Comment:
sklearn and bert versions used?

Reply:
sklearn version added. bert is a local `.py` file